src/main/java/org/terasology/taskSystem/actions/PerformTaskNode.java

```diff
@@ -215,32 +216,28 @@ private void completeTask(Actor oreon, TaskComponent taskComponent) {
                 if (buildingTemplate != null) {
                     List<SpawnBlockRegionsComponent.RegionToFill> relativeRegionsToFill = buildingTemplate.getParentPrefab().getComponent(SpawnBlockRegionsComponent.class).regionsToFill;
 
-                    List<Region3i> regionsToFill = new ArrayList<>();
+                    List<BlockRegion> regionsToFill = new ArrayList<>();
                     for (SpawnBlockRegionsComponent.RegionToFill regionToFill : relativeRegionsToFill) {
                         BlockRegion relativeRegion = regionToFill.region;
-                        BlockRegion absoluteRegion = relativeRegion.translate(JomlUtil.from(buildingComponent.centerLocation));
-                        regionsToFill.add(JomlUtil.from(absoluteRegion));
+                        BlockRegion absoluteRegion = relativeRegion.translate(buildingComponent.centerLocation, new BlockRegion(BlockRegion.INVALID));
```

Comment:
Don't use INVALID, make a copy of `regionToFill.region` instead and mutate in-place. (I'd like to avoid the usage of INVALID)